airflow-core/src/airflow/ui/src/i18n/locales/de/dashboard.json

```diff
@@ -0,0 +1,49 @@
+{
+  "welcome": "Willkommen",
+  "stats": {
+    "stats": "Statistiken",
+    "failedDags": "Fehlgeschlagene Dags",
+    "runningDags": "Laufende Dags",
+    "activeDags": "Aktive Dags",
+    "queuedDags": "Dags in Warteschlange"
+  },
+  "health": {
+    "health": "System-Gesundheit",
+    "metaDatabase": "Meta-Datenbank",
+    "scheduler": "Scheduler",
+    "triggerer": "Triggerer",
+    "dagProcessor": "Dag Prozessor"
+  },
+  "poolSlots": "Pool Plätze",
+  "managePools": "Pools Verwalten",
+  "history": "Historie",
+  "dagRun_one": "Dag Lauf",
+  "dagRun_other": "Dag Läufe",
+  "taskInstance_one": "Task Instanz",
+  "taskInstance_other": "Task Instanzen",
+  "assetEvent_one": "Asset-Ereignis",
+  "assetEvent_other": "Asset-Ereignisse",
+  "states": {
+    "queued": "Wartend",
+    "running": "Laufend",
+    "success": "Erfolgreich",
+    "failed": "Fehlgeschlagen",
+    "skipped": "Übersprungen",
+    "removed": "Gelöscht",
+    "scheduled": "Geplant",
+    "restarting": "Im Neustart",
+    "up_for_retry": "Warte auf neuen Versuch",
```

Comment:
Should it be "Wartet auf neuen Versuch" ? 
Since it describes the status, not the action of the task in the other cases 🤔 
same with "Wartet auf Neuplanung".

Reply:
Danke, das passt!